src/catalog/Cargo.toml

```diff
@@ -12,8 +12,8 @@ workspace = true
 
 [dependencies]
 api.workspace = true
-arrow.workspace = true
 arrow-schema.workspace = true
+arrow.workspace = true
```

Comment:
You can use `cargo +stable install taplo-cli --version ^0.9 --locked`.

IIRC taplo introduces some breaking changes between 0.8 and 0.9 on this `.workspace` case.

Reply:
https://github.com/tamasfe/taplo/pull/513 this one.